### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by DaleSeo**
The MIT License would be better than nothing for now.
It can be replaced later if we find a better option for us.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 [fullname]
```
Comment (DaleSeo) [inline LICENSE]: I need an official full name of the company to fill in this place holder.

**Comment by DaleSeo** [comment]
No worries! I will keep this PR open until we make a final decision.

File: LICENSE (hunk 1 above)
Comment (vuldin) [reply LICENSE]: I assume this would be Prodigy Education. @mattbullock you may know better.

Comment (mattbullock) [reply LICENSE]: This should be `Prodigy Education Inc.`

Comment (DaleSeo) [reply LICENSE]: Updated!

**Pull request merged**